src/View/index.js

```diff
@@ -0,0 +1,162 @@
+// =======================================================
+// View
+// =======================================================
+
+import React from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import PropTypes from 'prop-types'
+import forwardRefToWrappedComponent from '../utils/forwardRef'
+import generateResponsiveStyles from '../utils/generateResponsiveStyles'
+import r from '../../macro'
+import OIOResponsiveObjectPropType from '../utils/PropType'
+import withResponsiveObjectProps from '../utils/withResponsiveObjectProps'
+import withDynamicResponsiveProps from '../utils/withDynamicResponsiveProps'
+
+@forwardRefToWrappedComponent
+@withResponsiveObjectProps([
+   'display', 'float', 'position', 'top', 'left', 'right', 'bottom', 'scroll',
+   'flex', 'flexFlow', 'justifyContent', 'alignItems', 'order',
+   'height', 'width', 'maxHeight', 'maxWidth', 'minHeight', 'minWidth',
+   'margin', 'marginBottom', 'marginLeft', 'marginRight', 'marginTop',
+   'padding', 'paddingBottom', 'paddingLeft', 'paddingRight', 'paddingTop',
+   'paddingHorizontal', 'paddingVertical',
+   'border', 'borderBottom', 'borderLeft', 'borderRight', 'borderTop',
+   'backgroundColor', 'backgroundImage', 'backgroundPosition', 'backgroundSize',
+   'borderRadius', 'boxShadow', 'textAlign', 'zIndex'
+])
+
+@withDynamicResponsiveProps((props, breakpoint) => {
+   const { paddingHorizontal, paddingVertical, scroll } = props
+   const styleObject = {}
+
+   if (scroll && scroll[breakpoint] === 'on') {
+      styleObject.overflow = 'auto'
+      styleObject.WebkitOverflowScrolling = 'touch'
+   }
+
+   if (paddingHorizontal && paddingHorizontal[breakpoint]) {
+      styleObject.paddingLeft = paddingHorizontal[breakpoint]
+      styleObject.paddingRight = paddingHorizontal[breakpoint]
+   }
+
+   if (paddingVertical && paddingVertical[breakpoint]) {
```

Comment:
What if `paddingTop` and/or `paddingBottom` were explicitly set? Or `padding`?

Reply:
Updated docs with explanation